packages/devtool-extension/crx-plugin.ts

```diff
@@ -76,7 +76,7 @@ export class CRXPlugin {
         const data: Buffer[] = [];
         const output = await new Promise<Buffer>( (resolve, reject) => {
             stream.on('error', reject);
-            stream.on('data', chunk => data.push(chunk));
+            stream.on('data', chunk => data.push(typeof chunk === 'string'? Buffer.from(chunk):chunk));
```

Comment:
Could you please add spaces between condition and results `typeof chunk === 'string' ? Buffer.from(chunk) : chunk`